front/style/pages/_join.scss

```diff
@@ -0,0 +1,150 @@
+
+            @media (max-width: $tablet) {
+                top: 0;
+                transform: translateY(0);
+                margin: 0 auto;
+                width: 100%;
+            }
+
+            span {
+                @include text--large;
+                @include text--bold;
+                font-family: 'Gillsans';
+                border-bottom: 10px solid $yellow;
+                box-shadow: inset 0 -5px 0 $yellow;
+                display: inline-block;
+                padding: 0 10px;
+                height: 30px;
+
+                @media (max-width: $tablet) {
+                    height: 25px;
+                }
+            }
+
+            p {
+                text-align: justify;
+                padding: 2em;
+            }
+        }
+    }
+
+    &__form {
+        width: 65%;
+        background-color: $gray--white;
+        height: auto;
+        overflow: scroll;
+        padding: 10% 0;
+
+        @media (max-width: $tablet) {
+            width: 100%;
+            overflow: hidden;
+            padding: 5% 0;
+        }
+
+        textarea:focus,
+        input:focus,
+        select:focus {
+            outline: 0;
+            border: 1px solid $green--other;
+            transition: all 1s;
+        }
+
+        input,
+        select {
+            padding-left: 12px;
+            padding-right: 12px;
+            border-radius: 3px;
+            border: solid 1px $gray--light;
+            height: 60px;
+            box-sizing: border-box;
+            font-size: 14px;
+            font-family: "Roboto", Helvetica, sans-serif;
+        }
+
+        select#user_registration_birthdate_day,
+        select#become_adherent_birthdate_day,
+        select#user_registration_birthdate_month,
+        select#become_adherent_birthdate_month {
+            width: 90%;
+            margin-right: 10px;
+        }
+
+        select#user_registration_birthdate_year,
+        select#become_adherent_birthdate_year {
+            width: 90%;
+        }
+
+        .form__captcha {
+            width: 100%;
+        }
+
+        .register__form--full {
+            width: 100%;
+        }
+
+        .form__checkbox input[type="checkbox"]:checked+label:before {
+            transition: all 1s;
+            border-color: $green--other;
+            background: $green--other;
+        }
+
+        &__legal {
+            overflow: scroll;
+            border: 1px solid $gray--light;
+            border-radius: 3px;
+            padding: 1em;
+            margin-top: 30px;
+
+            @media (max-width: $tablet) {
+                margin-bottom: 0;
+            }
+        }
+
+        .btn {
+            border-radius: 3px;
+            max-width: 100%;
+            width: 100%;
+            color: $white;
+        }
+
+        .text__checkbox {
+            color: $gray--muted;
+
+            a {
+
+                color: $black--half;
+                border-bottom: 1px solid $black--half;
+            }
+        }
+    }
+}
```

Comment:
missing blank line at end of file : please configure your IDE to add automaticly a blank line at the end of files